runtime/fiber.c

```diff
@@ -75,10 +78,25 @@ struct stack_info** caml_alloc_stack_cache (void)
 
 Caml_inline struct stack_info* alloc_for_stack (mlsize_t wosize)
 {
+#ifdef __OpenBSD__
+  struct stack_info* si;
+  size_t len = sizeof(struct stack_info) +
+               sizeof(value) * wosize +
+               8 /* for alignment */ +
+               sizeof(struct stack_handler);
+  si =  mmap(NULL, len, PROT_WRITE | PROT_READ,
+             MAP_ANONYMOUS | MAP_PRIVATE | MAP_STACK, -1, 0);
+  if (si == MAP_FAILED)
+    return NULL;
+
+  si->size = len;
+  return si;
+#else
   return caml_stat_alloc_noexc(sizeof(struct stack_info) +
                                sizeof(value) * wosize +
                                8 /* for alignment */ +
                                sizeof(struct stack_handler));
```

Comment:
As a non-expert, I wonder what the "8 for alignment" is for. (Alignment of what with what?) Does this amount change if `stack_info` grows, as in this PR?

Reply:
This was primarily for arm64 16-byte alignment needs. Ive updated the comment to mention that explicitly.